packages/ubuntu_wsl_setup/integration_test/ubuntu_wsl_setup_test.dart

```diff
@@ -109,7 +109,11 @@ Future<void> testSelectYourLanguagePage(
   expectPage(tester, SelectLanguagePage, (lang) => lang.selectLanguageTitle);
 
   if (language != null) {
-    await tester.tap(find.widgetWithText(ListTile, language));
+    final tile = find.widgetWithText(ListTile, language, skipOffstage: false);
+    expect(tile, findsOneWidget);
+    await tester.scrollUntilVisible(tile, kMinInteractiveDimension);
```

Comment:
Humble question: `kMinInteractiveDimension` is inherited from `Material`? Wondering whether that could be too big or too small.

Reply:
Notice that it scrolls repeatedly until the target item becomes visible. This arbitrary number is just the delta for each scroll step, not the total scrolling distance. An arbitrary constant looked nicer than an arbitrary hardcoded value. :grin:

I recall that a dense ListTile has a height of 48 so this would be scrolling max one item at a time, but how would you like scrolling half the view at a time? Would that make it look less arbitrary?

```diff
diff --git a/packages/ubuntu_wsl_setup/integration_test/ubuntu_wsl_setup_test.dart b/packages/ubuntu_wsl_setup/integration_test/ubuntu_wsl_setup_test.dart
index 0744e81f..36fc79b8 100644
--- a/packages/ubuntu_wsl_setup/integration_test/ubuntu_wsl_setup_test.dart
+++ b/packages/ubuntu_wsl_setup/integration_test/ubuntu_wsl_setup_test.dart
@@ -111,7 +111,8 @@ Future<void> testSelectYourLanguagePage(
   if (language != null) {
     final tile = find.widgetWithText(ListTile, language, skipOffstage: false);
     expect(tile, findsOneWidget);
-    await tester.scrollUntilVisible(tile, kMinInteractiveDimension);
+    final viewRect = tester.getRect(find.byType(ListView));
+    await tester.scrollUntilVisible(tile, viewRect.height / 2);
     await tester.pump();
     await tester.tap(tile);
     await tester.pump();
```

Reply:
I liked that! Would you mind?

Reply:
Alright, let's go with that